site/app/Resources/views/notification/notification_inbox.html.twig

```diff
@@ -0,0 +1,77 @@
+{% extends 'layout.html.twig' %}
+
+{% block title %}
+    Notification Inbox
+{% endblock %}
+
+{% block pagepath %}Home &raquo; Notification Inbox{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('notification_inbox') }}
+{% endblock %}
+
+{% block content %}
+    <h1>Notification Inbox</h1>
+    <div class="inbox-header row">
+        <div class="col-xs-6 col-md-6 col-sm-6 inbox-title">
+            Your Notifications ({{ mgilet_notification_count(app.user) }})
+        </div>
+        <div class="col-xs-6 col-md-6 col-sm-6 text-right">
+            <a class="inbox-settings" href="/notification/settings">Notification Settings</a>
+        </div>
+    </div>
+    <ul class="all-notifications">
+        {# If there are any notifications #}
+        {% if (mgilet_notification_count(app.user)) %}
```

Comment:
notificationList should be empty in that case, so you don't need to add the special check.